project2/Vehicle/project2Uno.ino

```diff
@@ -0,0 +1,36 @@
+
+String commandFormat(String comm) {
```

Comment:
Why is this function not being used?

Reply:
This was from when I thought it wanted the full command printed as words